app/notifications/previous_notification.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class PreviousNotification < Noticed::Base
```

Comment:
Why previous ?

Reply:
It is meant to be all notifications under the `ActivityNotification` context, shall I use `OldNotifications` or `OtherNotifications`?

@tachyons